### DISCUSSION TITLE
Error when tryping to open the dll: "snAPI64.dll"

### DISCUSSION THREAD

**Discussion by evankumar** [PicoQuant/snAPI#10] [Q&A] [answered]
Hello,
After following the installation instructions from the guide, I am unable to open the snAPI dll from Demo_TimeTrace.py with the following error message:

 File "C:\Users\fastlab\Desktop\snAPI-1.0.2\snAPI-1.0.2\demos\.venv\Lib\site-packages\snAPI\Main.py", line 18, in <module>
    class snAPI:
  File "C:\Users\fastlab\Desktop\snAPI-1.0.2\snAPI-1.0.2\demos\.venv\Lib\site-packages\snAPI\Main.py", line 48, in snAPI
    dll = ct.WinDLL(os.path.abspath(os.path.join(os.path.dirname(__file__), '.\\snAPI64')))
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\fastlab\AppData\Local\Programs\Python\Python311\Lib\ctypes\__init__.py", line 376, in __init__
    self._handle = _dlopen(self._name, mode)
                   ^^^^^^^^^^^^^^^^^^^^^^^^^
OSError: [WinError 127] The specified procedure could not be found

I have ensured that I am running Python 11.7, Windows 10, and have installed the latest version of snAPI. Furthermore, I am able to open other DLLs without a problem.

Finally, when trying to execute this code from the Terminal (command line). I receive the following error: "The procedure entry point __std_tzdb_get_current_zone could not be located in the dynamic link library"

Please let me know if you have come across this issue or have a possible solution for this bug,

Thank you

**Comment by tpoint75** [comment, marked as answer]
Hello,

that sounds strange. I googled this and found following:
[https://community.citra-emu.org/t/the-procedure-entry-point-std-tzdb-get-time-zones-could-not-be-located-in-the-dynamic-link-library/900467](url)

It was solved by:
Update your C++ redist: [https://aka.ms/vs/17/release/vc_redist.x64.exe](https://aka.ms/vs/17/release/vc_redist.x64.exe) And make sure to run the latest Windows 10 version.

Please try it and tell us if it solved your issue,

By Torsten.

**Reply by evankumar** [reply]
Solved, thank you!
